src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +29,32 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {selectedTodo.id}
```

Comment:
It should look like in the example `Todo #2`, etc. That's the reason tests fail: 
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/86923089/ac046629-8cfc-4fe1-a074-ea654b19c2c9)